libscript/src/system.lcb

```diff
@@ -56,6 +56,34 @@ begin
 	MCSystemExecGetOperatingSystem(output)
 end syntax
 
+public foreign handler MCSystemExecGetArchitecture(out Architecture as String) returns nothing binds to "<builtin>"
+/**
+Summary:    The architecture
+
+Example:
+    if the architecture is "x86_64" then
+        -- Do something x86_64 specific
+    end if
+
+Description:
+Returns a string describing the instruction set architecture
+that is being used by the machine LiveCode is running on.
+The possible values are:
+
+* "x86" - 32-bit Windows, Mac and Linux
+* "x86_64" - 64-bit Windows, Mac and Linux
+* "i386" - iOS simulator
```

Comment:
Hmm.. we don't actually support the 32 bit simulator any more do we? Why wouldn't it return `x86` anyway?`x86_64` should note the iOS 64 bit simulator I think.

Reply:
Yes - we should choose either i386 or x86... And should it be 'x86-64' rather than 'x64_64'?

Reply:
I think the LCS form uses underscore

Reply:
It does, but we could change it in LCB

Reply:
I would keep it as close as possible to the arch portion of the platform ID spec if we can. I'm not sure if we have the info required for arm variants though.